modules/rosa-create-objects.adoc

```diff
@@ -109,78 +107,99 @@ Create a new cluster.
 
 |--private-link
-| Specifies to use AWS PrivateLink to provide private connectivity between VPCs and services. The `--subnet-ids` argument is required when using `--private-link`.
+|Specifies to use AWS PrivateLink to provide private connectivity between VPCs and services. The `--subnet-ids` argument is required when using `--private-link`.
+
+|--region <region_name>
+|The name of the AWS region where your worker pool will be located, for example, `us-east-1`. This argument overrides the `AWS_REGION` environment variable.
+
+|--role-arn <arn>
+|The Amazon Resource Name (ARN) of the installer role that {cluster-manager} uses to create the cluster. This is required if you have not already created account roles.
 
-|--region
-|The AWS region (string) where your worker pool will be located. This argument overrides the `AWS_REGION` environment variable.
+|--service-cidr <address_block>
+a|Block of IP addresses (ipNet) for services, for example, `172.30.0.0/16`.
 
-|--role-arn string
-|The Amazon Resource Name (ARN) of the installer role that {cluster-manager} will assume to create the cluster.
+[IMPORTANT]
+====
+OVN-Kubernetes, the default network provider in {product-title} 4.11 and later, uses the `100.64.0.0/16` IP address range internally. If your cluster uses OVN-Kubernetes, do not include the `100.64.0.0/16` IP address range in any other CIDR definitions in your cluster.
+====
 
-|--service-cidr
-|Block of IP addresses (ipNet) for services. Example: `172.30.0.0/16`
+a|--sts \| --non-sts
+|Specifies whether to use AWS Security Token Service (STS) or IAM credentials (non-STS) to deploy your cluster.
 
-|--subnet-ids
-|The subnet IDs (string) to use when installing the cluster. Subnet IDs must be in pairs with one private subnet ID and one public subnet ID per availability zone. Subnets are comma-delimited. Example: `--subnet-ids=subnet-1,subnet-2`. Leave the value empty for installer-provisioned subnet IDs.
+|--subnet-ids <aws_subnet_id>
+|The AWS subnet IDs to use when installing the cluster, for example, `subnet-01abc234d5678ef9a`. Subnet IDs must be in pairs with one private subnet ID and one public subnet ID per availability zone. Subnets are comma-delimited, for example, `--subnet-ids=subnet-1,subnet-2`. Leave the value empty for installer-provisioned subnet IDs.
 
 When using `--private-link`, the `--subnet-ids` argument is required and only one private subnet is allowed per zone.
 
 |--support-role-arn string
 |The Amazon Resource Name (ARN) of the role used by Red Hat Site Reliabilty Engineers (SREs) to enable access to the cluster account to provide support.
 
-|--version
-|The version (string) of {product-title} that will be used to install the cluster or cluster resources, including `account-role`. Example: `4.12`
+|--version string
+|The version of {product-title} that will be used to install the cluster or cluster resources, including `account-role`, for example, `4.12`.
```

Comment:
The version for account-roles is like X.Y but the version used for cluster creation should include the Z stream, like X.Y.Z.

Reply:
I've given two separate examples for clarity here.